sdks/python/apache_beam/runners/portability/stager_test.py

```diff
@@ -723,6 +765,65 @@ def test_remove_dependency_from_requirements(self):
     self.assertEqual(['apache_beam\n', 'avro-python3\n', 'numpy\n'],
                      sorted(lines))
 
+  def _create_file(
+      self, requirements_file, temp_dir, populate_cache_with_sdists):
+    if not populate_cache_with_sdists:
+      self.create_temp_file(os.path.join(temp_dir, 'nothing.whl'), 'Fake whl')
+      self.create_temp_file(
+          os.path.join(temp_dir, 'nothing.tar.gz'), 'Fake tarball')
+    else:
+      self.create_temp_file(
+          os.path.join(temp_dir, 'nothing.tar.gz'), 'Fake tarball')
```

Comment:
If we download whls and if a whl is not present on PyPI, we download sdist. That is the reason why it is present in both branches.

Reply:
I meant that you can add `nothing.tar.gz` outside the `if` since this code is in both branches. But feel free to keep as is if you think it's more clear this way.

Reply:
Done